File: C/Day_15.c
// Getting a three digit number anad printing their sum of digits
#include <stdio.h>

int main() {
    int num, result_1,result_2,result_3,result;
    printf("Enter  three digit number:");
    scanf("%d",&num);
    result_1 = num %100;
    result_2 = num /100;
    result_3 = num %10;
    result = result_1/10;
    printf("The final result is : %d \n",result_2+result_3+result);
    return 0;
}

File: Python/Day_15.py
#Getting three digit number and printing their sum of the digits
num = int(input("Enter a three digit number:"))
result = num // 100
result_1 = num%10
result_2 = num%100
result_3 = result_2//10
Final_result = result + result_1 + result_3
print("The final result is :",Final_result)
